src/modules/navigator/rtl_params.c

```diff
@@ -110,10 +110,10 @@ PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
  * Fly straight to the return location or planned mission landing and land there or
  * use the planned mission to get to those points.
  *
- * @value 0 Return home via direct path
- * @value 1 Return to a planned mission landing, if available, via direct path, else return to home via direct path
- * @value 2 Return to a planned mission landing, if available, using the mission path, else return to home via the reverse mission path
- * @value 3 Return via direct path to closest destination: home, mission landing pattern or safe point
+ * @value 0 Return to closest safe point (home or rally point) via direct path.
+ * @value 1 Return to closest safe point (planned mission landing or rally point), if available, else return to home.
```

Comment:
1. What is the benefit of ignoring home as a possible landing destination? ie. why would you choose to go to a rally point that was defined but not to home? Also the wording little odd it implies "home is not a safe point".
2. The path taken is important - ie direct. 
3. I like the term "mission landing pattern"

So depending on your answer to 1 - one of:

> * @value 1 Return to closest safe point (start of mission landing pattern, rally point, or home), via direct path. If safe point is a landing pattern then land.

> * @value 1 Return to closest safe point other than home (mission landing pattern or rally point), via direct path. If no mission landing or rally points are defined return home via direct path.

Reply:
Or to put it another way, why do we need both valy 1 and value 2?

Reply:
Home should only be considered for value 1 if no landing or rally point is defined because with a VTOL/FW it's safer. It the user doesn't set a rally point, then 1 results in the same behavior as is currently implemented.

Your wording sounds better, changed that (for value 1)

Reply:
> Or to put it another way, why do we need both valy 1 and value 2?

You mean between TYPE 1 and 2? That it follows the mission, either forward (mission landing) or inverse (home).